### ISSUE TITLE
Build and style BGS group page

### ISSUE THREAD

**Issue by angievanhegan** [MantaRayMedia/bgs#134]

**Comment by cutlern** [comment]
PHP in the twitter hashtag view needs Arseni to look at it

**Comment by cutlern** [comment]
@meldera I think Nadia's the best to ask

**Comment by meldera** [comment]
Nothing wrong with the view... No tweets with a hashtag   -   no output.

**Comment by cutlern** [comment]
@meldera does the view work with retweets? Because we retweeted RSTMH yesterday with #zika, so zika should be pulling in at least one tweet but the view is showing nothing?

**Comment by russ-manta** [comment]
@cutlern have you tried running cron?  Cron need to run to pul the tweets in - worth checking that

**Comment by meldera** [comment]
@cutlern don't think id does. cron job is working.

**Comment by angievanhegan** [comment]
@cutlern Are you working on this one?

**Comment by cutlern** [comment]
@angievanhegan I passed the twitter issue back to Arseni, but if it's working like Arseni says then I think it can be put into blocked because I think this is still blocked by the organisation civi stuff

**Comment by angievanhegan** [comment]
@cutlern Some of this is blocked by design in #154 (mostly for the cards) but I think there's lots you can do in the meantime. Thanks very much.

**Comment by angievanhegan** [comment]
@cutlern Design is now ready for you to use. See link above

**Comment by angievanhegan** [comment]
@cutlern Did I add the curated stuff at the bottom? There only needs to be three sections on the page: events, resources, news.

**Comment by cutlern** [comment]
@angie I added them cause i'm doing some work on the content feeds, they need to be built with fields instead of teasers so that they can link to filter pages that are already filtered by the group

**Comment by pipe-manta** [comment]
@angievanhegan You can check all the tasks, but need all the Civi staff and Events card/landing page to be finished.

**Comment by angievanhegan** [comment]
@pipe-manta Would you be able to tick all the boxes on here to show what's been done?  Thanks.

**Comment by angievanhegan** [comment]
@ican-icant So, are we putting a pink line at the top of call the cards on this page? Rather than different colours for each of the four types? Thanks

**Comment by angievanhegan** [comment]
(comment from  @StephMSlack) @angievanhegan - what order do the items appear in? Do I select what appears on the page or is it automatic? If automatic, there is the possibility that blog overwhelms any policy information relevant to the group, even though the policy information is older. If the editor can choose the content displayed & in what order, then combining is fine, but if automatic, then we might need to keep the blog separate or think about other ways that important content could be displayed

@StephMSlack My feeling is that we're overthinking/speccing this.

**Comment by angievanhegan** [comment]
@ican-icant Are we putting the pink lines at the top of all these cards?

**Comment by ican-icant** [comment]
@angievanhegan I would

**Comment by ican-icant** [comment]
@angievanhegan do I need to do anything else?

**Comment by angievanhegan** [comment]
@pipe-manta I think there's still some styling tweaks for the cards: 

![screen shot 2018-03-02 at 09 08 37](https://user-images.githubusercontent.com/3796718/36902699-68618398-1df9-11e8-8fad-547b55fac6d6.png)
![screen shot 2018-03-02 at 09 08 44](https://user-images.githubusercontent.com/3796718/36902700-68809774-1df9-11e8-8842-4685a316aea3.png)

**Comment by pipe-manta** [comment]
@angievanhegan Resources are done (fixed high as on the other places), but Latest events feed need to be done after Event's landing page

**Comment by angievanhegan** [comment]
@pipe-manta As per the wireframe, this should be able to show a twitter feed at the top. The box above is ticked so I think it's already done, I just can't see how to add it. Please can you add one to show me? Thank you very much.

**Comment by StephMSlack** [comment]
@angie just looking at the groups page - we will need to be able to add content/images potentially beyond the feeds. Can the additional or top content have the standard content nuggets enabled as well (and possibly the box links?)
The type of content could be:

Mini article specific to the SIG - eg the Diabetes SIG has a session at the BGS conference in April, here is what we are discussing and why, we need your help on X, dont forget to book your ticket to the conference. etc.
Call for members to feed into a consultation, where the consultation request requires some explanatory text and information to download
report on an event that has been attended by SIG member (which would be too specific for a general news article)

**Comment by angievanhegan** [comment]
@StephMSlack Isn't this exactly why we've created the member alert on a group page?

**Comment by angievanhegan** [comment]
Angie to check the twitter feed. 
Issue 2 still needs doing.

**Comment by angievanhegan** [comment]
@StephMSlack The event cards and the officers (coming in from Civi) will be part of other tickets. Can the rest of this be signed off now? Thanks

**Comment by StephMSlack** [comment]
@angievanhegan - cant seem to add a twitter feed. Am I missing something?

**Comment by angievanhegan** [comment]
@dilwarhussain can you show @StephMSlack and I how to add a twitter feed to this page please? Thanks

**Comment by StephMSlack** [comment]
@dilwarhussain the content block at the top should have the show more function, but this doesn't seem to be working at the moment.

**Comment by dilwarhussain** [comment]
@StephMSlack 

- The 'show more' is dependant on the number of characters in the body text. We have the limit at 600, above that the 'show more' button will show. Working fine here: http://bgs-dev.mrmdev.co.uk/bgs-scotland
- We also have the twitter hashtag view working, it's currently using the Manta Ray Media twitter feed. See here: http://bgs-dev.mrmdev.co.uk/falls-and-bone-health
- What would you like the title of the twitter hashtag to be?

**Comment by StephMSlack** [comment]
thanks @dilwarhussain. 

I've added a twitter hashtag to http://bgs-dev.mrmdev.co.uk/peri-operative-care-of-older-people-undergoing-surgery-pops but it doesn't seem to be showing up. Am I missing a step?

Can we set the name of the twitter feed ourselves, as it may change depend on the group?

Also, the blog feed doesnt seem to be displaying correctly on http://bgs-dev.mrmdev.co.uk/peri-operative-care-of-older-people-undergoing-surgery-pops. I mentioned it in task #183

**Comment by angievanhegan** [comment]
@StephMSlack I think you've created it correctly. It may be a problem with the hashtag as #malaria works fine. @dilwarhussain What do you think?

**Comment by dilwarhussain** [comment]
@angievanhegan @StephMSlack The current way this works is to look at hashtags from one specific account. In this case it's currently looking at Manta Ray, when we get BGS's twitter info we can authenticate and it will only look at BGS twitter. 

To allow this view to look at multiple accounts and hashtags on different pages would require further development.

**Comment by StephMSlack** [comment]
sorry @dilwarhussain I didn't specify which join button I was talking about (definitely not helpful). I meant the join the group button within the page. Screenshot below when logged in and then logged out (when the join the group button doesn't seem to be visible). 
![screen shot 2018-04-24 at 15 26 04](https://user-images.githubusercontent.com/32127102/39193765-2bbb20ac-47d4-11e8-8b49-5839fa3b9686.png)
![screen shot 2018-04-24 at 15 26 27](https://user-images.githubusercontent.com/32127102/39193771-2dcd8fec-47d4-11e8-867d-b018653a2473.png)

**Comment by dilwarhussain** [comment]
@StephMSlack Ah right sorry, I had just been working on the Join link in the header and just assumed it was regarding that. I'll take a look at the RHS join link.

**Comment by StephMSlack** [comment]
thanks @dilwarhussain 
@angievanhegan this is working now
